csharp/dotnet/security/use_deprecated_cipher_algorithm.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Security.Cryptography;
+					
+public class Encryption
+{
+	public void CreateAes1() {
+		//ok: use_deprecated_cipher_algorithm
+		var key = Aes.Create();
+	}
+	
+	public void CreateAes2() {
+		//ok: use_deprecated_cipher_algorithm
+		var key = Aes.Create("ImplementationName");
+	}
+
+	public void CreateRijndael1() {
+		//ok: use_deprecated_cipher_algorithm
+		var key = Rijndael.Create();
```

Comment:
Rijndael seems to be deprecated based on the docs: https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.rijndael?view=net-6.0

Reply:
Rijndael is basically AES, so this is not a relevant security risk either. The same holds, as for 3DES.

Reply:
👍